src/styles/index.scss

```diff
@@ -129,7 +124,7 @@ header > div {
   background-color: theme("colors.beige.darker");
   width: 56px;
   height: 56px;
-  margin-left: 24px;
+  // margin-left: 24px;
```

Comment:
detail: remove the comment if it's unecessary

Reply:
oh yeah missed that thanks